### ISSUE TITLE
Multiple words names

### ISSUE THREAD

**Issue by canvasplay** [yavorskiy/comment-parser#41]
Hello guys!
I need some advice. I've tried to figure it out myself with no luck, maybe its not supported...

The question is:
How could I define a name with multiple words? p.e.:
```
/**
* @test {type} My Module
*/
```
I would like to be parsed as 
```
{
  name: "My Module",
  ...
}
```

Unfortunaltely I can only get the name parsed like that using the optional delimiters `[My Module]`.
I understand that probably there are no languages suporting variable names with multiple words... 
In my use case I want to define a doclet  as kind of "section" to hold other doclets, I would like to keep the description part... p.e.:
```
/**
* @section Brand Colors - Here you can find all the brand colors...
*/
```
Which is the best bet?
1. Assume `@section` name is always `tag.name+' '+tag.description`.
2. Use the optional delimiter and assume the optional property is senseless for `@section` tags.
3. Custom parser using the `-` delimiter?
3. ...

What do yout think? Am I missing something?

Thanks in advance.
Cheers.

**Comment by yavorskiy** [comment]
I am sorry it's far too late response but you could do 
```
@section [Brand Colors] Here you can find all the brand colors
```
it's just a side-effect of how optional parameters are parsed. Making use of quotes sounds very reasonable though, I will try to implement it

**Comment by yavorskiy** [comment]
implemented literal names in `comment-parser@0.5.4`
